src/Native/Unix/configure.cmake

```diff
@@ -29,6 +30,13 @@ endif ()
 # We compile with -Werror, so we need to make sure these code fragments compile without warnings.
 set(CMAKE_REQUIRED_FLAGS -Werror)
 
+# This compiler warning will fail code as innocuous as:
+# static pthread_mutex_t lock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
Does this warning trigger a lot of failures in the repo or just this one? I'm wondering if there's any value in trying to adhere to it. That said, I'm also fine disabling it.

Reply:
Oops. I should have included a list to begin with. Here's the entire list from `./build.sh`: https://gist.github.com/omajid/308d684538daaef24590b4c90f1aebb9

Reply:
I see, thanks, I agree with disabling it, especially given those dependencies we don't own.